lib/delayed/backend/worker.rb

```diff
@@ -9,6 +10,8 @@ class << self
       def configure
         yield(config)
 
+        # debugger
```

Comment:
Please remove commented out debugger.

Reply:
Hey Matt. Progress on this is halted at the moment because, as I was trying to fix the specs thinking that they were broken due to the v2 upgrade, I realized they don't and/or shouldn't even pass on the develop branch.

When I do a fresh clone of develop + bundle install they blow up before reaching the first test unless ActiveRecord is updated to at least `4.2`.  Additionally, the shared examples from delayed job itself do not pass.  Both of these issues are discussed somewhat on PR #18.

Thus this is in very WIP state at the moment with many leftover debuggers and commented-out lines.  I intend to chat with @fpcyan about this when she returns from PTO, and maybe pick up on this PR again after that.  At this point I feel that the v2 SDK functionality should be working fine, but would not rly be comfortable until specs actually pass.

Reply:
Gotcha.  Thanks for the update!